### DISCUSSION TITLE
Welcome to INSIDE-ARG Discussions!

### DISCUSSION THREAD

**Discussion by twinysam** [twinysam/INSIDE-ARG#5] [General]
Giving this a try, it might come in handy for people who want to make suggestions or ask questions about the ARG or the documentation.

So if you have a question or an idea, fire away!

**Comment by sarahalobaidli** [comment]
Can i have full code of this game

**Reply by twinysam** [reply]
If you are talking about INSIDE, it's not an open source game. If you are referring to the ARG, it doesn't really have much code but most of it can be found in the terminal41 website: https://github.com/twinysam/INSIDE-ARG/tree/master/terminal41.link
